src/stream/src/executor/sink.rs

```diff
@@ -218,12 +219,33 @@ impl<F: LogStoreFactory> SinkExecutor<F> {
         if self.sink.is_sink_into_table() {
             processed_input.boxed()
         } else {
+            let labels = [
+                &actor_id.to_string(),
+                "NA", // TODO: remove the connector label for log writer metrics
```

Comment:
Any reason to erase the connector label?

We can get the connector label easily from `self.sink`, which can be supported by implementing the following method for `Sinkimpl`

```
impl SinkImpl {
    pub fn sink_name(&self) -> &'static str {
        fn get_sink_name<S: Sink>(_sink: &S) -> &'static str {
            S::SINK_NAME
        }

        dispatch_sink!(self, sink, get_sink_name(&**sink))
    }
}
```

Reply:
Can do. I was thinking that, for Log**Writer**Metrics, it's supposed to be unaware of  the log reader i.e. the actual Sink implementation.

Also, the `sink_name` is here, which I think is enough for users.